molecule/pxc/playbooks/prepare.yml

```diff
@@ -102,4 +113,37 @@
       service:
         name: firewalld
         state: stopped
-      when: ansible_distribution == "OracleLinux" and ansible_distribution_major_version >= "8"
+      when: (ansible_os_family == "RedHat") and (ansible_distribution_major_version|int == 8 or ansible_distribution_major_version|int == 9)
+
+    - name: Setup SSH keys CentOS
+      authorized_key:
+        user: centos
+        key: "{{ lookup('file', 'public_keys') }}"
+        state: present
+        exclusive: False
+      when: (ansible_distribution == "CentOS") and (lookup('env', 'DESTROY_ENV') == "no")
```

Comment:
We do not have DESTROY_ENV var, why do we have this condition here? We always destroy instances at this moment, please remove keys upload.